templates/html/patient_information.html

```diff
@@ -45,11 +45,17 @@
                   <tr>
                     <td class="inner centered" style="padding:0;padding:10px;text-align:center;">
                       <p class="h1 content-width" style="font-size:14px;line-height:1.5;margin:0;margin-bottom:10px;font-weight:bold;margin-left:auto;margin-right:auto;max-width:400px;">
-                        {{.SignupHello}}
-                  </p>
+                        {{.PatientHeadline}}
```

Comment:
shall we name these ones "PatientInfoXXX" ?

Reply:
yep, makes sense to follow the same patterns for all items